scripts/environment.sh

```diff
@@ -83,6 +83,8 @@ else
   export UPGRADED_RAW_IMAGE_NAME="CENTOS_9_NODE_IMAGE_K8S_${UPGRADED_K8S_VERSION}-raw.img"
 fi
 
+export LIVE_ISO_IMAGE="https://artifactory.nordix.org/artifactory/metal3/images/iso/minimal_linux_live-v2.iso"
```

Comment:
I would prefer this to be in e2e-config file since it is needed only by the e2e. I am usually export vars here only if it is necessary but if the var is used only by e2e framework would be better to be in the e2e file